### ISSUE TITLE
[BUG] Breaks the "Reload UI" functionality (via the top-right button in "Settings")

### ISSUE THREAD

**Issue by God-damnit-all** [namkazt/sd-webui-prompt-history#37]
For some reason, when using this extension, restarting the UI via the "Reload UI" button in "Settings" doesn't work:

![image](https://github.com/namkazt/sd-webui-prompt-history/assets/53661808/4198d602-143d-4f32-ac5d-a4c3f27553b8)

It simply displays `Restarting UI...` in the console and hangs forever, not even Ctrl+C will interrupt it.

I've also tested running the server with `DEBUG` verbosity; no additional information about why reload UI could've frozen was displayed in the console.

"Apply and restart UI" in the Extensions tab _does_ still work, that effectively restarts the entire server though.

**Comment by Moorer624** [comment]
Yes. I have the same exact problem. The problem has existed since the moment I started using this extension. M1 Mac Studio.
